core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceSettingService.scala

```diff
@@ -126,7 +127,12 @@ class WorkspaceSettingService(protected val ctx: RawlsRequestContext,
           case SeparateSubmissionFinalOutputsSetting(SeparateSubmissionFinalOutputsConfig(_)) => None
           case UseCromwellGcpBatchBackendSetting(UseCromwellGcpBatchBackendConfig(_))         => None
           case PubliclyReadableSetting(PubliclyReadableConfig(_))                             => None
-          case CompactDataTablesSetting(CompactDataTablesConfig(_))                           => None
+          case CompactDataTablesSetting(CompactDataTablesConfig(enabled)) =>
+            if (!enabled) {
+              Some(validationErrorReport(setting.settingType, "this setting cannot be disabled once enabled"))
+            } else {
+              None
+            }
```

Comment:
If a user has no `CompactDataTablesSetting` at all, then adds a setting with enabled=false, will this error still throw? That is somewhat misleading.